src/pages/[platform]/build-a-backend/storage/authorization/index.mdx

```diff
@@ -117,25 +127,25 @@ The following policy would allow authenticated users full access to files with a
 export const storage = defineStorage({
   name: 'myProjectFiles',
   access: (allow) => ({
-    'foo/{entity_id}/*': [
+    'folder/{entity_id}/*': [
       // {entity_id} is the token that is replaced with the user identity id
       allow.entity('identity').to(['read', 'write', 'delete'])
     ]
   })
 });
 ```
 
-A user with identity id "123" would be able to perform read/write/delete operations on files within `foo/123/*` and would not be able to perform actions on files with any other prefix.
+A user with identity id "documents" would be able to perform read/write/delete operations on files within `folder/documents/*` and would not be able to perform actions on files with any other prefix.
```

Comment:
I don't undersstand this. The word "identity ID" and and "entity id" seems to be intermixed. What would a "identity id" of "documents" mean? Isn't this always a user sub?